.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -3,13 +3,23 @@ set -euxo pipefail
 
 source .buildkite/scripts/common.sh
 
+# Override the agent package version using a string with format <major>.<minor>.<patch>
+# NOTE: use only after version bump when the new version is not yet available, for example:
+# OVERRIDE_AGENT_PACKAGE_VERSION="8.10.3"
+OVERRIDE_AGENT_PACKAGE_VERSION=""
+
+if [[ -n "$OVERRIDE_AGENT_PACKAGE_VERSION" ]]; then
+  OVERRIDE_TEST_AGENT_VERSION=${OVERRIDE_AGENT_PACKAGE_VERSION}"-SNAPSHOT"
+else
+ OVERRIDE_TEST_AGENT_VERSION=""
+fi
 # PACKAGE
-AGENT_PACKAGE_VERSION=8.10.2 DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64 PACKAGES=tar.gz mage package
+AGENT_PACKAGE_VERSION="${OVERRIDE_AGENT_PACKAGE_VERSION}" DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64 PACKAGES=tar.gz mage package
 
 # Run integration tests
 set +e
 # Use 8.10.2-SNAPSHOT until the first 8.10.3-SNAPSHOT is produced.
-AGENT_VERSION="8.10.2-SNAPSHOT" TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 TEST_INTEG_CLEAN_ON_EXIT=true SNAPSHOT=true mage integration:test
+AGENT_VERSION="${OVERRIDE_TEST_AGENT_VERSION}" TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 TEST_INTEG_CLEAN_ON_EXIT=true SNAPSHOT=true mage integration:test
```

Comment:
The comment above this can be removed since it references 8.10.2 and 8.10.3 where that isn't relevant anymore.